util/keypair.test.ts

```diff
@@ -7,8 +7,10 @@ import {
     pemToArrayBuffer,
     SerializedBuffer,
     pemToJSONBuffer,
+    privateKeyFromBuffer,
+    fingerPrintPublicKeyFromPrivateKey,
```

Comment:
think this needs to be `fingerprintPublicKeyFromPrivateKey` per the auto-suggestion